cms/src/test/resources/test-data/api-specifications/ApiSpecificationMetadataJcrRepositoryTest/apiSpecMetadataRoot_withActualMetadata.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+# Root API Specifications metadata node under module config node
+# /hippo:configuration/hippo:modules/api-specification-sync/api-specification-sync/hippo:moduleconfig
+
+- path: /hippo:configuration/hippo:modules/api-specification-sync/hippo:moduleconfig/api-specification-metadata
+  primaryType: hipposys:moduleconfig
+
+# Metadata of API Specification A
+- path: /hippo:configuration/hippo:modules/api-specification-sync/hippo:moduleconfig/api-specification-metadata/api-spec-a-6c684d98-61e9-4ff6-bf40-5e288bd692f8
+  jcr:primaryType: hipposys:moduleconfig
```

Comment:
is there any purpose to the `jcr` prefix being dropped in the other test metadata?

Reply:
Another good one!

Yes, it's actually the entries without the `jcr:` prefix that are more correct, as this is what class uk/nhs/digital/test/util/MockJcrRepoProvider.java:45 expects.